src/main/java/com/sarahhand/json/JsonReaderWriter.java

```diff
@@ -61,4 +63,35 @@ public JsonReaderWriter(PropertyNamingStrategy pns){
 			log.error("Error writing JSON to stream.", e);
 		}
 	}
-}
+
+	/** This creates an object out using a JSON file.
```

Comment:
Because of the previous error. I can't see if my corrections worked, so I need to wait until that error is resolved in order to proceed.